example/ffmpeg.js

```diff
@@ -1,32 +1,44 @@
 // Install fluent-ffmpeg before running this!
 const readline = require('readline');
-const path   = require('path');
-const fs     = require('fs');
-const ytdl   = require('..');
+const path = require('path');
+const fs = require('fs');
 const ffmpeg = require('fluent-ffmpeg');
+const ytdl = require('..');
 
-const url = 'https://www.youtube.com/watch?v=TGbwL8kSpEk';
-const audioOutput = path.resolve(__dirname, 'sound.mp4');
+const url = 'https://www.youtube.com/watch?v=qgRUr-YUk1Q';
+const audioOutput = path.resolve(__dirname, 'sound.m4a');
 const mainOutput = path.resolve(__dirname, 'output.mp4');
 
 console.log('downloading audio track');
 ytdl(url, {
-  filter: format => format.container === 'm4a' && !format.encoding
-}).on('error', console.error)
-  .on('progress', (chunkLength, downloaded, total) => {
+  filter: format => format.container === 'mp4' && !format.qualityLabel
+})
+  .on('error', console.error)
+  .on('progress', (_, downloaded, total) => {
```

Comment:
even if unused in this example, having `chunkLength` can help users understand the `progress` event just a little bit more.  it's more important to be over descriptive in example code.